src/Link.tsx

```diff
@@ -19,6 +19,7 @@ export interface LinkOwnProps {
   replace?: boolean;
   target?: string;
   to: string;
+  className: string;
```

Comment:
Just a nitpick: But I like to sort interface fields alphabetically I wouldn't require you to do this but if you touch this anyway I would be great if you could change this.